lib/main/STM32H7/Middlewares/ST/STM32_USB_Device_Library/Class/CDC_HID/Inc/usbd_cdc_hid.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Author: Chris Hockuba (https://github.com/conkerkh)
+ *
+ */
+
```

Comment:
Almost exactly a year ago PR #5742 changed the license blocks to mention both betaflight and cleanflight. 

Although maybe that was only stuff in src?

Reply:
I find F7 code also has the exact same block.

There are two problems with this.

(1) A file created inside the library.
(2) Old license.

Reply:
@jflyper: Good point. Refactoring this into a less contentious approach seems trivial: #8115. Maybe do the same for H7?